README.md

```diff
@@ -53,14 +53,9 @@ to Read the Docs.
 
 If you don't want to make edits with a CLI and code editor, you can use the GitHub UI:
 
-1. Click the "Edit" button for the document you want to change:
-
-<img width="1265" alt="image" src="https://user-images.githubusercontent.com/17532695/199124407-0f2036e8-c669-40f4-9083-d23e636b553d.png">
-
-2. Make you're changes.
-3. Select "Create a **new branch** for this comment and start a pull request", add a commit message and select "Commit changes":
-   <img width="1216" alt="image" src="https://user-images.githubusercontent.com/17532695/199124752-e7c0eccd-1300-4c27-9821-82b6155b4c9e.png">
-
-4. Tag someone to review your changes.
+1. Click the "Edit" button for the document you want to change: <img width="1265" alt="image" src="https://user-images.githubusercontent.com/17532695/199124407-0f2036e8-c669-40f4-9083-d23e636b553d.png">
+1. Make your changes.
+1. Select "Create a **new branch** for this comment and start a pull request", add a commit message and select "Commit changes": <img width="1216" alt="image" src="https://user-images.githubusercontent.com/17532695/199124752-e7c0eccd-1300-4c27-9821-82b6155b4c9e.png">
+1. Tag someone to review your changes.
```

Comment:
Any idea why these are all "1."s? The build creates a numbered list from 1 to 4.

Reply:
I agree it reads weird, but the idea here is that markdown is keeping track of the numbering so you don't have to. This is [one of mdformat's opinions](https://mdformat.readthedocs.io/en/stable/index.html)

Reply:
Oh ok 👍